conf/test.config

```diff
@@ -31,4 +31,3 @@ params {
   // Options
   skip_cool = true
```

Comment:
Is this documented what it means?

Reply:
It was in the help message, but I just added a part in the usage.md file.